tools/Release/CreatePullRequest.ps1

```diff
@@ -0,0 +1,48 @@
+<#
+.PARAMETER versionPath
+  The path to the msbuild props file where the version number is specified
+#>
+
+Param(
+  [string]
+  $versionPath
+)
+
+if ($versionPath -eq "")
+{
+  $versionPath = "$PSScriptRoot\..\CustomMSBuild\Versioning.props"
+}
+
+$versions = New-Object xml
+$versions.PreserveWhitespace = $true
+$versions.Load($versionPath)
+foreach ($propertyGroup in $versions.Project.PropertyGroup)
+{
+  if ($propertyGroup.VersionRelease -ne $null)
+  {
+    $versionNumber = $propertyGroup.VersionMajor.'#text' + "." + $propertyGroup.VersionMinor.'#text' + "." + $propertyGroup.VersionBuildNumber.'#text'
+
+    break;
+  }
+}
+
+if ($versionNumber -eq $null)
+{
+  Write-Error "No version number was found in $versionPath."
+  Exit
+}
+
+$branchName = "releases/$versionNumber"
```

Comment:
Is there any change we need to do to ADO?
What is fundamentally different from what we current do by running a nightly build on master?

Reply:
This script creates the branch + pull request that is used to update the version numbers. Once that PR is merged, then the nightly build will run on the newly created commit. 

There is no change that needs to happen in ADO.